app/app.gradle.kts

```diff
@@ -18,6 +18,7 @@ dependencies {
     compile(group = "com.google.code.gson", name = "gson", version = "2.8.1")
     fun testFx(name: String, version: String = "4.0.+") =
         create(group = "org.testfx", name = name, version = version)
+    runtime(group = "edu.wpi.first.ntcore", name = "ntcore-jni", version = "+", classifier = "all")
```

Comment:
Also we have a whole release pipeline. What it does is start from the bottom (wpiutil now), and builds/publishes it. Then builds the next layer up, then up then up. These are versions that are published to release rather then development, so the versions are completely different, even though they have the same as the latest development release.